CONTRIBUTING.md

```diff
@@ -93,15 +93,15 @@ We are two modules for helping us structure our commit message in a clean, undes
 
 Please keep in mind that the commit messages will be included in the release notes, therefore it is important that you use the proper way of creating a git commit message.
 
-After staging your changes with `git add`, run `npm run commit` to start the interactive commit message CLI.
+After staging your changes with `git add`, run `git cz` to start the interactive commit message CLI.
```

Comment:
How does ```git cz``` get added as an alias?

Reply:
As part of the global npm install. Pretty neat